server/MyDonor/MyDonor.Domain/Models/Appointments.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
```

Comment:
Move the `using System.ComponentModel.DataAnnotations.Schema;` import to global usings.